notebooks/00-release-28.0.0/03-lookup-tables-28.ipynb

```diff
@@ -0,0 +1,1129 @@
+    "FROM lookup.\"example-flights-airportsizes\"\n",
+    "GROUP BY 1\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "075619ae-7691-44a2-8ed3-7216142c3091",
+   "metadata": {},
+   "source": [
+    "### Return lookup values using a JOIN\n",
+    "\n",
+    "JOINs can be performed between lookup and standard tables.\n",
+    "\n",
+    "Run the following cell to see the results of a JOIN between the lookup table you created and the `example-flights-lookup` table you ingested earlier."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c21db4c7-1052-4cfb-80d0-3d2557cdf9ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "SELECT\n",
+    "    b.v AS \"airportSize\",\n",
+    "    COUNT(DISTINCT a.Origin) AS \"airports\",\n",
+    "    COUNT(*) AS \"flights\",\n",
+    "    SUM(a.Distance) AS \"totalDistance\"\n",
+    "FROM \"example-flights-lookup\" a\n",
+    "LEFT JOIN lookup.\"example-flights-airportsizes\" b ON a.Origin = b.k\n",
+    "WHERE TIME_IN_INTERVAL(__time,'2005-11-30T11:00:00/2015-11-30T08:00:00')\n",
+    "GROUP BY 1\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b35b59e-503d-4a00-bf49-97781f9756d7",
+   "metadata": {},
+   "source": [
+    "### Return lookup values using a function\n",
+    "\n",
+    "Use the LOOKUP [string function](https://druid.apache.org/docs/latest/querying/sql-scalar#string-functions) to dereference keys to values in an SQL query directly. This function is available as both a [SQL](https://druid.apache.org/docs/latest/querying/sql-functions#lookup) and [native](https://druid.apache.org/docs/latest/querying/math-expr#string-functions) expression.\n",
+    "\n",
+    "In the following cell, the `Origin` key from the `example-flights-lookup` table is used to match against the `\"example-flights-airportsizes\"` lookup, returning the value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e58653fa-54a9-439a-8461-b95e49ce9daa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "SELECT\n",
+    "    LOOKUP(\"Origin\", 'example-flights-airportsizes') AS \"airportSize\",\n",
+    "    COUNT(DISTINCT Origin) AS \"airports\",\n",
+    "    COUNT(*) AS \"flights\",\n",
+    "    SUM(Distance) AS \"totalDistance\"\n",
+    "FROM \"example-flights-lookup\"\n",
+    "WHERE TIME_IN_INTERVAL(__time,'2005-11-30T11:00:00/2015-11-30T08:00:00')\n",
+    "GROUP BY 1\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8bbb8830-c0e9-45d7-9533-4f0c65144367",
+   "metadata": {},
+   "source": [
+    "Use the overloaded form of the LOOKUP function to return a value even when there is no matching key.\n",
+    "\n",
+    "> This version of the LOOKUP function requires Druid version 28.0.0 or above."
```

Comment:
Already noted in the requirements

Reply:
This is the point where I didn't know what was best. The rest of the notebook works fine in Druid 27 (and I dare say previous versions) - it's just this one cell that doesn't because of the overload.

Ie, do we want to say this notebook works on Druid 27, but that this cell requires Druid 28? Or that this notebook is for Druid 28, which gives the impression none of it works on older versions?

@techdocsmith